kermit/src/commonMain/kotlin/co/touchlab/kermit/Logger.kt

```diff
@@ -164,16 +167,19 @@ open class Logger(
             LoggerGlobal.defaultConfig.logWriterList = logWriter.toList() + LoggerGlobal.defaultConfig.logWriterList
         }
 
-        fun setDefaultTag(tag: String) {
-            LoggerGlobal.defaultConfig.defaultTag = tag
+        fun setTag(tag: String) {
```

Comment:
I would say it would have same level of confusion as `defaultTag`. Someone would feel there is `root` tag and `non-root` tag. Also meaning of `root` might be confusing. We don't use `root` anywhere else.